Sources/Plasma/FeatureLib/pfLocalizationMgr/pfLocalizationDataMgr.cpp

```diff
@@ -542,16 +545,15 @@ void LocalizationDatabase::IMergeData()
 
 void LocalizationDatabase::IVerifyElement(const ST::string &ageName, const ST::string &setName, LocalizationXMLFile::set::iterator& curElement)
 {
-    std::vector<ST::string> languageNames;
-    ST::string defaultLanguage;
+    std::unordered_set<ST::string, ST::hash> languageNames;
```

Comment:
```suggestion
    std::unordered_set<ST::string> languageNames;
```

`std::hash` is already specialized for `ST::string`